### ISSUE TITLE
Query do not supports several chained parameters on the same resource

### ISSUE THREAD

**Issue by lemsy** [Aidbox/Issues#120]
We have the **SearchParameters** `target` and `received` and 
make the query `GET fhir/Task?based-on:RequestGroup.target:ServiceRequest.specimen:Specimen.received=ge2019-05-12&based-on:RequestGroup.target:ServiceRequest.specimen:Specimen.received=le2019-07-02` but we obtain the error:

```
resourceType: OperationOutcome
id: exception
text: {status: generated, div: 'ERROR: table name "based_on_requestgroup" specified more than once'}
issue:
- {severity: fatal, code: exception, diagnostics: 'ERROR: table name "based_on_requestgroup" specified more than once'}

```

**Comment by niquola** [comment]
On which version of Aidbox do you have this error. Can you test with 0.4.5?
By FHIR spec this query will not give you a response which you expect. 
It will  be two independent join sequences for each chain.

Probably _filter can solve this problem - hope we will release it in 0.4.6. 
For now, you can switch to SQL on FHIR and custom query - https://docs.aidbox.app/api/custom-search#parameters-in-query

**Comment by lemsy** [comment]
We are testing in 0.4.5-SNAPSHOT.

> By FHIR spec this query will not give you a response which you expect.
> It will be two independent join sequences for each chain.

Yes, but still exists the error of allow to use the chained params only once.

**Comment by niquola** [comment]
Fixed in 0.4.6-SNAPSHOT

**Comment by niquola** [comment]
I've added a custom modifier for your case in 0.4.6-SNAPSHOT

```
/Task?based-on:RequestGroup.target:ServiceRequest.specimen:Specimen.received:btw=2019-05-12,2019-07-02 
```
